sentry-android-distribution/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <uses-permission android:name="android.permission.INTERNET" />
+
+  <application>
+    <provider
+      android:name="io.sentry.android.distribution.DistributionContentProvider"
+      android:authorities="${applicationId}.sentry-distribution-provider"
```

Comment:
Looks like we can make this a bit shorter: https://github.com/getsentry/sentry-java/blob/c8125f3c9015a841a3830b4544ea9f86ae7b4f76/sentry-android-core/src/main/AndroidManifest.xml#L13

- Just `.SentryDistributionProvider` for name
- `${applicationId}.SentryDistributionProvider` for android:authorities to match prtg